src/main/java/DockerImage.java

```diff
@@ -106,6 +106,22 @@ public String getRepository() {
         return this.environmentVariables;
     }
 
+    /**
+     * Returns the value of the specified environment variable if it is configured for this Docker image,
+     * else returns a null.
```

Comment:
can we return an empty string ? Everywhere we used this method, we don't really check if its null.

Reply:
The reason I chose to return a NULL is because, there can be env variables with empty values, which will be mapped to an empty string.

Reply:
Got it. Makes sense. Do you think we need to have NPE checks when we use env variables?

Reply:
Added null checks to retrieved env variables.